papas/datatypes/Cluster.h

```diff
@@ -35,15 +35,26 @@ class Cluster {
                 When creating a merged cluster it should ideally be set to the total eneergy of the cluster
       */
   Cluster(const Cluster& cluster, unsigned int index, IdCoder::ItemType type, char subtype = 'u', float val = 0.0);
+
+  /** Constructor: makes new merged cluster
+   @param[in]  overlappingClusters list of clusters to be merged, they must have same type and subtype and must be
+   simple
+               unmerged clusters.
+   @param[in]  index of the collection into which the cluster is to be stored
+   @param[in]  subtype subtype of cluster to be created eg 'm' for merged,
+   */
+  Cluster(std::list<const Cluster*> overlappingClusters, unsigned int index, char subtype = 'm');
```

Comment:
I don't understand, it's certainly possible to create a shared_ptr out of an existing object. 
Can you send me a link to the code where the overlappingClusters list is built?